docs/docs/10.1-animation.md

```diff
@@ -86,7 +87,7 @@ You'll notice that animation durations need to be specified in both the CSS and
 
 `ReactCSSTransitionGroup` provides the optional prop `transitionAppear`, to add an extra transition phase at the initial mount of the component. There is generally no transition phase at the initial mount as the default value of `transitionAppear` is `false`. The following is an example which passes the prop `transitionAppear` with the value `true`.
 
-```javascript{3-5}
```

Comment:
Wow that's impressive!

I'll go ahead and put back the ranges on the lines that should be emphasized.